storage/reads/datatypes/storage_common.proto

```diff
@@ -24,6 +24,12 @@ service Storage {
 
   rpc Hints (google.protobuf.Empty) returns (HintsResponse);
 
+  // TagKeys performs a read operation for tag keys
+  rpc TagKeys (TagsRequest) returns (stream TagsResponse);
+
+  // TagValues performs a read operation for tag values
+  rpc TagValues (TagsRequest) returns (stream TagsResponse);
```

Comment:
Add `TagValuesRequest` that has the same parameters as `TagKeysRequest` with the addition of `string tag_key` to specify which tag key the values should be produced for